app/components/ItuneCard/tests/index.test.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *
+ * Tests for ItuneCard
+ *
+ */
+
+import React from 'react';
+import { renderWithIntl } from '@utils/testUtils';
+import ItuneCard from '../index';
+
+describe('<ItuneCard />', () => {
+  it('should render and match the snapshot', () => {
+    const { baseElement } = renderWithIntl(<ItuneCard />);
+    expect(baseElement).toMatchSnapshot();
+  });
+
+  it('should contain 1 ItuneCard component', () => {
+    const { getAllByTestId } = renderWithIntl(<ItuneCard />);
+    expect(getAllByTestId('itune-card').length).toBe(1);
+  });
+
+  it('should render itune song details inside the card', () => {
+    let artistName = 'eminem';
+    let trackName = 'Till I Collapse (feat. Nate Dogg)';
```

Comment:
can we please create an object with the song details in describe and then access it using keys in the test

Reply:
done